README.md

```diff
@@ -61,6 +61,8 @@ This [dbt](https://github.com/dbt-labs/dbt) package provides compatibility with
 | Schema tests          | sequential_values             | :white_check_mark:    | no customization needed                                                |
 | Schema tests          | unique_combination_of_columns | :white_check_mark:    | no customization needed                                                |
 | Schema tests          | accepted_range                | :white_check_mark:    | no customization needed                                                |
+| Schema tests          | at_least_one                | :white_check_mark:    |  custom macro provided                                               |
+| Schema tests          | expression_is_true             | :white_check_mark:    |  custom macro provided                                               |
```

Comment:
Correct indentation

Reply:
These will be converted into table, indentation won't matter much, Mohan.